docs/release-notes/8.18.asciidoc

```diff
@@ -0,0 +1,152 @@
+.Rules cannot be enabled if they're corrupted while upgrading from 7.17.x to 8.x 
+[%collapsible]
+====
+*Details* +
+If rule saved objects were corrupted when you upgraded from 7.17.x to 8.x, you may run into an error when turning on your rules. 
+
+*Workaround* +
+
+Duplicate your rules and enable them.
+
+====
+// end::known-issue[]
+
+// tag::known-issue[]
+[discrete]
+.The technical preview badge incorrectly displays on the alert suppression fields for event correlation rules 
+[%collapsible]
+====
+*Details* +
+On April 8, 2025, it was discovered that alert suppression for event correlation rules is incorrectly shown as being in technical preview when you create a new rule. For more information, check (https://github.com/elastic/docs-content/issues/1021)[#1021].
+
+====
+// end::known-issue[]
+
+[discrete]
+[[deprecations-8.18.0]]
+==== Deprecations
+* The user and host risk score modules are being deprecated ({kibana-pull}202775[#202775]).
+* The following SIEM signal migration endpoints were deprecated ({kibana-pull}202662[#202662]):
+
+** POST /api/detection_engine/signals/migrations
+** DELETE /api/detection_engine/signals/migrations
+** POST /api/detection_engine/signals/finalize_migrations
+** GET /api/detection_engine/signals/migration_status
+
+[discrete]
+[[features-8.18.0]]
+==== New features
+* Provides automatic migration for detection rules to help convert existing SIEM rules into Elastic equivalents.
+* The Automatic Import functionality is now generally available ({kibana-pull}208523[#208523]).
+* Adds in-text citations to AI assistant responses whenever fact providers (such as the knowledge base or alert information) are used to generate the response ({kibana-pull}206683[#206683]).
+* Allows you to https://github.com/elastic/kibana/issues/174168[customize prebuilt rules]. You can modify most rule parameters, export and import prebuilt rules — including customized ones — and upgrade prebuilt rules while retaining customization settings ({kibana-pull}212761[#212761]).
+* Adds initial support for the service entity type in the Entity Store, whereas previously, only user and host entity types were supported ({kibana-pull}207336[#207336], {kibana-pull}206582[#206582], {kibana-pull}206268[#206268], {kibana-pull}202344[#202344]).
+* Allows you to configure how often the enrich policy runs for the entity store ({kibana-pull}207374[#207374], {kibana-pull}204437[#204437]).
+* Provides configuration options to the entity store through additional API parameters ({kibana-pull}206421[#206421]).
+* Introduces a status tab to the entity store management page ({kibana-pull}201235[#201235]).
+* Allows you to install and reinstall entity stores from the Engine Status page ({kibana-pull}208149[#208149]).
+* Introduces ways to monitor and fix gaps in rule executions, which can lead to missed alerts or reduced rule coverage ({kibana-pull}206313[#206313]).
+* The manual runs functionality is now generally available ({kibana-pull}209535[#209535]).
```

Comment:
Does it make sense to group the GA functions one after the other instead of lines apart?

Reply:
I don't have any issue with that but it'd break the current organization principle, which is to group changes based on product area.